ui/src/app/utils/runtime-utils.tsx

```diff
@@ -263,10 +272,14 @@ const compareDiskSize = (
   newRuntime: RuntimeConfig
 ): RuntimeDiff => {
   let desc = 'Disk Size';
-  let diffType;
+  let diffType: RuntimeDiffState;
   if (newRuntime.diskSize < oldRuntime.diskSize) {
     desc = 'Decease ' + desc;
-    diffType = RuntimeDiffState.NEEDS_DELETE_RUNTIME;
+    if (newRuntime.detachableDisk && oldRuntime.detachableDisk) {
```

Comment:
just to check on detachableDisk field in this PR. will be null for dataproc? Would that work everything? 
In the previous PR, it has GCE or Dataproc, and only execute PD logic for GCE VM.

Reply:
Good question. That was my expectation, but checking the code again - there was some potential for divergence when computeType changed. I've fixed this issue (more thorough testing to come in the following PR where I add controls for PD and PD type)